guidance/llms/_model.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from loguru import logger
```

Comment:
I don't think using loguru in a library is a good idea, especially the upstream repo does not rely on loguru. Instead we might just use the plain `logging.getLogger` function, or refer to how the upstream does.

(Please check <https://github.com/Delgan/loguru/issues/349#issuecomment-707267009>)

Reply:
Builtin logging has been used instead of loguru..